OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -61,6 +61,7 @@
     private static final String PUSH_NOTIFICATIONS_FLAG = "PUSH_NOTIFICATIONS";
     private static final String WHITE_LIST_OF_DOMAINS = "WHITE_LIST_OF_DOMAINS";
     private static final String API_URL_VERSION = "API_URL_VERSION";
+    private static final String YOUTUBE_VIDEO = "YOUTUBE_VIDEO";
```

Comment:
@saeedbashir thoughts?

Reply:
I think the iOS app is also using the same config name. am I right? @saeedbashir

Reply:
The configurations are not for the player but for the youtube videos. `YOUTUBE_VIDEO` seems a better name.

Reply:
Since we don't have a parent level config dedicated for video handling, having the config's name as `YOUTUBE_VIDEO` doesn't make any sense.

I would recommend we go with the following config:

```
YOUTUBE_IN_APP_PLAYER:
   ENABLED: true
   API_KEY: 'api_key_goes_here'
```

Reply:
done.